apps/prairielearn/assets/scripts/instructorQuestionSettingsClient.ts

```diff
@@ -11,6 +19,46 @@ onDocumentReady(() => {
   );
   const saveButton = document.querySelector<HTMLButtonElement>('#save-button');
 
+  new TomSelect('#topic', {
+    valueField: 'name',
+    searchField: ['name'],
```

Comment:
Should we allow searching of the description as well?

Reply:
Good idea. Added description to the `searchField` in [98f2dd5](https://github.com/PrairieLearn/PrairieLearn/pull/10868/commits/98f2dd558c3f286228530e2a7899b8db990394be).